go/base_client.go

```diff
@@ -417,9 +422,16 @@ func (client *baseClient) executeBatch(
 	var spanPtr uint64
 	otelInstance := GetOtelInstance()
 	if otelInstance != nil && otelInstance.shouldSample() {
-		// Pass the request type to determine the descriptive name of the command
-		// to use as the span name
-		spanPtr = otelInstance.createBatchSpan()
+		// Check if there's a parent span in the context
+		if parentSpanPtr := otelInstance.extractSpanPointer(ctx); parentSpanPtr != 0 {
+			// Create child batch span with parent
+			// Since we don't have create_batch_otel_span_with_parent, we create a named child span
+			// using the parent span pointer to establish the parent-child relationship
+			spanPtr = otelInstance.createBatchSpanWithParent(parentSpanPtr)
+		} else {
+			// Create independent batch span (current behavior)
```

Comment:
```suggestion
			// Create independent batch span
```
Once the code is merged, this makes no sense.

Reply:
Done